classesExamples/Fruit.py

```diff
@@ -0,0 +1,30 @@
+class Fruit(object):
+
+    def __init__(self):
+        print('constructor in Fruit class')
+
+    def nutrition(self, fruit):
+        fruit_nutrition = {'banana': 4, 'orange': 5, 'exotic': 6}
```

Comment:
Churra look here, snake case 🐍change it too 🐪
```suggestion
        fruitNutrition = {'banana': 4, 'orange': 5, 'exotic': 6}
```

Reply:
I did it in the code, because I'm not sure if I should commit this suggestion, how this works?

Reply:
Yes you can commit the suggestion, and Github will create a commit for with it for you. However it's good that you did it on the code, because that particular variable was being used more than once. When using this tool we should try to be aware of the impact of it, and specially if further refactor will be required with it, always letting the authors know about it.